seaworthy/tests-core/test_logs.py

```diff
@@ -2,14 +2,14 @@
 import struct
 import threading
 import unittest
+from collections import namedtuple
```

Comment:
😱

Reply:
Easier than making a whole new class just to put one attribute on it. :-/